randovania/interface_common/options.py

```diff
@@ -112,6 +112,7 @@ def from_json(cls, value: dict) -> "PerGameOptions":
     "advanced_timeout_during_generation": Serializer(identity, bool),
     "auto_save_spoiler": Serializer(identity, bool),
     "dark_mode": Serializer(identity, bool),
+    "experimental_settings": Serializer(identity, bool),
```

Comment:
Add a new version/migration in `persisted_options`, just so newer rdv releases uses a different config file to older. (and then using an old version won't revert this setting randomly).